### PR TITLE
Animated login

### PR DESCRIPTION
**Pull request by Mantra-7**
Made login form with animated submit button.

### CODE REVIEW

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]
Everything is Ok. Just change the font-family if you have any other choice. and add the Readme file. Get the idea of how to designe the Readme from others project

**Review by saikatmu31 — APPROVED** [review]
Just add a Readme File

File: Login_Forms/login-animated/style.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+#login_box .submit .border{
+    padding: 2px 2px;
+    height: 100%;
+    width: 100%;
+    position: relative;
+    overflow: hidden;
+}
+
+#login_box .submit #login_box_submit{
```
Comment (thepranaygupta) [inline Login_Forms/login-animated/style.css:60]: add some margin to the left and top of the submit button
image

**Review by thepranaygupta — APPROVED** [review]

**Comment by Mantra-7** [comment]
Changes done. Please check again.

**Review by thepranaygupta — APPROVED** [review]

**Pull request merged**